.github/workflows/build_test_deploy.yml

```diff
@@ -509,6 +509,12 @@ jobs:
       - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run: ./scripts/publish-native.js $GITHUB_REF
       - run: ./scripts/publish-release.sh
+      - run: git update-index --no-skip-worktree package.json
+      - run: yarn
```

Comment:
Seems like this could potentially fail if npm is not updated yet? Should I use a script to wait and retry a few times?